source/_posts/rituals.md

```diff
@@ -14,7 +14,11 @@ Alternate between Wednesdays 1pm and Fridays 9am (Lima time).
 ### Office hours
 
 Monday and thursday 4-6pm (Lima time).
+Office hours can be proposed by everyone in favor to help with our colleagues.
```

Comment:
Hi @memecast, thanks for the PRs :yellow_heart: 

Can you please add this change directly to the main [README.md](https://github.com/Laboratoria/developers.laboratoria.la/blob/master/README.md) and send a PR?

Thanks!